app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt

```diff
@@ -17,8 +17,7 @@ import com.mapbox.services.android.navigation.v5.navigation.NavigationConstants
 import com.mapbox.services.android.navigation.v5.utils.DistanceFormatter
 import com.mapbox.services.android.navigation.v5.utils.LocaleUtils
 import com.mapbox.vision.VisionManager
-import com.mapbox.vision.core.utils.SystemInfoUtils
-import com.mapbox.vision.core.utils.snapdragon.SupportedSnapdragonBoards
```

Comment:
Should we remove it? Will it work without the dependencies?

Reply:
No, I'll fix